adapters/huaweiads/huaweiadstest/supplemental/bad_resquest_incorrect_huawei_adtype3.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
Nitpick: typo in filename. Can you change `bad_resquest_` to `bad_request_`?

Reply:
done, thank you